holidays/countries/sint_maarten.py

```
@@ -0,0 +1,115 @@
+
+        # Sint Maarten Day.
+        self._add_holiday_nov_11(tr("Sint Maarten Day"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Christmas Day"))
+
+        # Second day of Christmas.
+        self._add_christmas_day_two(tr("Second Day of Christmas"))
+
+
+class SX(SintMaarten):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)